lms/djangoapps/organizations/serializers.py

```diff
@@ -26,3 +26,14 @@ class Meta:
         fields = ('url', 'id', 'name', 'display_name', 'contact_name', 'contact_email', 'contact_phone',
                   'logo_url', 'created', 'modified')
         read_only = ('url', 'id', 'created',)
+
+
+class SimpleOrganizationSerializer(serializers.ModelSerializer):
```

Comment:
What is the reason for creating another serializer instead of using `BasicOrganizationSerializer`?